### PR TITLE
add techleads to codeowners

### PR DESCRIPTION
**Pull request by forummlcb**

### CODE REVIEW

**Review by phamcong — CHANGES_REQUESTED** [review]
/modules/web_app @phamcong

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1,8 @@
-@tiepvupsu @tnlong1997
+@tiepvupsu @tnlong1997 @hailoc12
```
Comment (forummlcb) [inline CODEOWNERS:1]:
```suggestion
@tiepvupsu @tnlong1997 @hailoc12 @seanphan
```

File: CODEOWNERS [hunk 2]
```diff
@@ -1 +1,8 @@
...
+
+/modules/crawlers/ @hailoc12
+/modules/data_wranglers/
+/modules/legacies/
+/modules/ml/ @victurtle
+/modules/ml_api/ @victurtle
+/modules/web_app/
```
Comment (forummlcb) [inline CODEOWNERS:8]:
```suggestion
/modules/web_app/ @phamcong
```

**Comment by forummlcb** [comment]
> /modules/web_app @phamcong

done

**Pull request merged**